src/core/tests/Test_ZypperPackageManager.py

```diff
@@ -154,7 +154,7 @@ def test_package_manager(self):
 
         # test for get_dependent_list
         # legacy_test_type ='Happy Path'
-        dependent_list = package_manager.get_dependent_list("man")
+        dependent_list = package_manager.get_dependent_list(["man"])
```

Comment:
@rane-rajasi  - it's possible for a function to process a singleton and a list as input. The caller needn't worry about the structure. Change should have been flagged and corrected.